doc/api_methods.md

```diff
@@ -500,13 +514,17 @@ The layer itself.
 
 #### Arguments
 
-- **auth_token:** string
+Arguments | Description
+--- | ---
+auth_token | string
+
 
 ### layer.setParams(_key, value_)
 
-Sets the configuration of a layer when using [named maps](http://docs.cartodb.com/cartodb-platform/maps-api.html#named-maps-1). It can be invoked in different ways:
+Sets the configuration of a layer when using [named maps](/cartodb-platform/maps-api/named-maps/). It can be invoked in different ways:
+
+#### Arguments
```

Comment:
This is an example (heading says Arguments) and I'd move it after the "Returns" subsection.